plain_hasher/util/plain_hasher/Cargo.toml

```diff
@@ -1,13 +0,0 @@
-[package]
-name = "plain_hasher"
-description = "Hasher for 32-bit keys."
-version = "0.1.0"
-authors = ["Parity Technologies <admin@parity.io>"]
-license = "MIT"
-keywords = ["hash", "hasher"]
-homepage = "https://github.com/paritytech/plain_hasher"
-
-[dependencies]
-crunchy = "0.1.6"
-ethereum-types = "0.3"
-hashdb = { version = "0.2.0", path = "../hashdb" }
```

Comment:
what happened with this file? ;)

Reply:
It was a left-over from when I moved over the code from `parity-ethereum` – I must have messed up the `git subtree` command at some point.